packages/shared/src/view/Layout.js

```diff
@@ -1,48 +1,22 @@
-// @flow
+// @flow strict
 
 import * as React from 'react';
-import { View, StatusBar } from 'react-native';
-
-import StyleSheet from '../PlatformStyleSheet';
-import Device from '../Device';
+import { StatusBar } from 'react-native';
 
 type Props = {|
-  children: React.Node,
+  +children: React.Node,
 |};
 
-const createStyles = () => {
-  const styles: Object = {
-    outerWrapper: {
-      flex: 1,
-      flexDirection: 'row',
-      justifyContent: 'center',
-    },
-    innerWrapper: {
-      flex: 1,
-    },
-  };
-
-  if (Device.isWideLayout()) {
-    styles.innerWrapper.maxWidth = Device.getWideDeviceThreshold();
-  }
-  return StyleSheet.create(styles);
-};
-
-/**
- * This is base Layout component with default width. It can expand up to
- * portrait mode width. This means that the content is centered in
- * landscape mode.
- */
-export default function Layout({ children }: Props) {
-  const defaultStyle = createStyles();
+export default function Layout(props: Props) {
```

Comment:
Is there a reason not to use destructuring here?

Reply:
Is there a reason to use destructing here? :)

Reply:
Fair enough. I just prefer destructuring :smile:

Reply:
I see - that's very subjective and it should not be part of the code review... :)

Reply:
You modified it because you preferred not to destructure. So I am just asking why you make the change as it seems to be arbitrary.

Reply:
Aha, I actually created this component from scratch but GitHub thinks I just modified it because I used the same filename with different component (I actually renamed Layout to LayoutSingleColumn...). That's why.